src/operator/activation-inl.h

```diff
@@ -1,198 +0,0 @@
-/*
```

Comment:
It's kind of hard to review the changes since github diff doesn't recognize the files have been renamed when there're too many. Will a smaller PR generate a more readable diff?

Reply:
break up this pull request?

Reply:
nvm

Reply:
According to @reminisce `git mv` will make git aware the file you moved and gives you a much better/readable diff

Reply:
I agree. I will probably need to redo the refactor and use git mv to move files.

Reply:
I tried `git mv` and rebased the entire history, but git still shows files are removed and added back.